src/Symfony/Component/Form/Extension/Core/Type/DateTimeType.php

```diff
@@ -251,6 +251,7 @@ public function configureOptions(OptionsResolver $resolver)
             'empty_data' => function (Options $options) {
                 return $options['compound'] ? array() : '';
             },
+            'input_format' => 'Y-m-d H:i:s',
```

Comment:
Should we set the default here or rely on the transformer default ? IMHO, it's better here so the form type and the transformer are more decoupled.